ch4-03-translations.md

```diff
@@ -40,6 +40,7 @@ contributions under the same license.
 - 🇬🇷 [Chapter I][gr-philosophy], [Chapter II][gr-economics], and [Chapter III][gr-technology] is available in Greek thanks to [nikos].
 - 🇮🇷 [Chapter I, Chapter II, and Chapter III][fa-all] is available in Persian thanks to [Nima].
 - 🇮🇩 [Chapter I][id-philosophy] and [Chapter II][id-economics] is available in Indonesian thanks to [Hazmi Tri Laksono][hazmi]
+- 🇮🇹 Almost [the whole site][it-site], lessons inclueded, is available in Italian thanks to [gabridome]
```

Comment:
typo: inclueded -> included

Reply:
I absolutely agree. I have just to figure out how to correct the PR and I will do.